packages/ui/src/lib/Textarea.svelte

```diff
@@ -84,8 +103,10 @@
 		}
 	});
 
-	let maxHeight = fontSize * 1.5 * maxRows + padding.top + padding.bottom;
-	let minHeight = fontSize * 1.5 * minRows + padding.top + padding.bottom;
+	const lineHeight = 1.6;
+
+	let maxHeight = fontSize * maxRows + padding.top + padding.bottom;
+	let minHeight = fontSize * minRows + padding.top + padding.bottom;
```

Comment:
These depend on reactive properties (from `$props`) so these ought to be `$derived` statements

Reply:
danke! Fixed